repository-data/application/src/main/resources/hcm-config/namespaces/website.cnd

```diff
@@ -3,25 +3,19 @@
 <'hippostd'='http://www.onehippo.org/jcr/hippostd/nt/2.0'>
 <'hippostdpubwf'='http://www.onehippo.org/jcr/hippostdpubwf/nt/1.0'>
 <'common'='http://digital.nhs.uk/jcr/common/nt/1.0'>
-<'hippotranslation'='http://www.onehippo.org/jcr/hippotranslation/nt/1.0'>
 <'hippotaxonomy'='http://www.hippoecm.org/hippotaxonomy/nt/1.2'>
+<'hippotranslation'='http://www.onehippo.org/jcr/hippotranslation/nt/1.0'>
 <'hippogallery'='http://www.onehippo.org/jcr/hippogallery/nt/2.0'>
 
-[website:basedocument] > hippo:document, hippostd:publishableSummary, hippostdpubwf:document
-  orderable
-  - common:searchable (boolean) = 'true' autocreated
-
-[website:financial] > hippostd:relaxed, hippotranslation:translated, website:basedocument
-  orderable
-
 [website:aboutapi] > hippo:compound, hippostd:relaxed
   orderable
 
 [website:address] > hippo:compound, hippostd:relaxed
   orderable
 
-[website:hubnewsandevents] > hippostd:relaxed, hippotaxonomy:classifiable, hippotranslation:translated, website:basedocument
```

Comment:
Please do not remove any entries within the CND.
Any documents already created with that document type will break

Reply:
:thumbsup: Not quite sure how that happened but great catch.

Reply:
@ShaneAhern-IRE I'm not able to revert this removal without causing a 503. Is there a good way of trying to debug that?